05-classification.Rmd

```diff
@@ -253,8 +254,12 @@ attrs <- c("Perimeter", "Concavity")
 my_distances <- table_with_distances(cancer[, attrs], new_point)
 neighbours <- cancer[order(my_distances$Distance), ]
 ```
-To predict the label of a new observation, i.e., classify it as either benign
-or malignant, the K-nearest neighbour classifier generally finds the $K$
+
+In order to actually make predictions for new observations in practice, we
+will need a classification algorithm; here we will use the K-nearest neighbour
+classification algorithm.
+To predict the label of a new observation (here, classify it as either benign
+or malignant), the K-nearest neighbours classifier generally finds the $K$
 "nearest" or "most similar" observations in our training set, and then uses
 their diagnoses to make a prediction for the new observation's diagnosis. To
```

Comment:
Should we define what K is here? For example, we could follow that sentence with: "$K$ in this algorithm is number of nearest neighbours, and is a value we choose (we will discuss how we choose this later in the chapter)."

Reply:
edited